musig/src/tests.rs

```diff
@@ -0,0 +1,342 @@
+#[cfg(test)]
```

Comment:
This is already a `tests` module, so you can wrap it into `#[cfg(test)]` in `lib.rs` and not declare an additional module here.